Add new BBEs on binding patterns (#3850)
ballerina-platform/ballerina-distribution: examples/rest-binding-pattern-in-list-binding-pattern/rest_binding_pattern_in_list_binding_pattern.md

# Rest binding pattern in list binding pattern

> If the number of binding patterns is less than the members in the list binding pattern, it creates a new list value consisting of the members that are not matched, which is called a rest binding pattern (`...r`).

Reviewer: ```suggestion
If the number of binding patterns is less than the members in the list value, you can use the rest binding pattern (`...r`), which creates a new list value consisting of the members that are not bonded.
```
